spec/models/orphan_spec.rb

```diff
@@ -437,9 +437,10 @@
           it 'generate csv content for givens orphans' do
             orphan = build_stubbed :orphan, name: "John", gender: "Male", father_given_name: "Mark", family_name: "Doe"
             orphan_attrs = orphan.as_json(methods: [:full_name, :father_name])
-            orphan_attrs.merge!({ partner_name: "partner name" })
+            orphan_attrs[:partner_name] = "partner name"
+            orphan_attrs[:current_sponsor_for_csv] = []
             orphan = OpenStruct.new(orphan_attrs)
-            output = "Osra Num,Full Name,Father Name,Date Of Birth,Gender,Province Name,Partner Name,Father Is Martyr,Father Deceased,Mother Alive,Priority,Status,Sponsorship Status,Current Sponsor\n,John Mark Doe,Mark Doe,#{orphan.date_of_birth},Male,#{orphan.province_name},partner name,#{orphan.father_is_martyr},#{orphan.father_deceased},#{orphan.mother_alive},#{orphan.priority},#{orphan.status},#{orphan.sponsorship_status},--\n"
+            output = "Osra Num,Full Name,Father Name,Date Of Birth,Gender,Province Name,Partner Name,Father Is Martyr,Father Deceased,Mother Alive,Priority,Status,Sponsorship Status,Current Sponsor,Sponsor OSRA Num\n,John Mark Doe,Mark Doe,#{orphan.date_of_birth},Male,#{orphan.province_name},partner name,#{orphan.father_is_martyr},#{orphan.father_deceased},#{orphan.mother_alive},#{orphan.priority},#{orphan.status},#{orphan.sponsorship_status}\n"
```

Comment:
I think the only reason it might work is because they are at the end, but what if they were eliminated in the middle, like
```
fooA,fooB,fooC\n
1,2,3\n
4,6\n
7,8,9\n
```